subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt

```diff
@@ -19,16 +19,13 @@ import com.avito.utils.logging.CILogger
 import com.avito.utils.logging.StubCILogger
 import com.avito.utils.logging.commonLogger
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.Timeout
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
-import java.util.concurrent.TimeUnit.SECONDS
 
 internal class TestExecutorIntegrationTest {
```

Comment:
Please, make an issue about testing in minikube/kind/... like environment.
It will be easier to cover edge-cases more hermetically.